packages/angular_devkit/schematics/tools/file-system-engine-host-base.ts

```diff
@@ -321,10 +339,27 @@ export abstract class FileSystemEngineHostBase implements
       return factory();
     }
 
+    if (this._scheduler.has(name)) {
+      return observableOf(<T>(options: T | undefined, context: SchematicContext) => {
+        if (options !== undefined && !json.isJsonObject(options)) {
+          throw new UnregisteredTaskException(name);
+        }
+        const job = this._scheduler.schedule(
+          name,
+          options ? { ...JSON.parse(JSON.stringify(options)) } : context,
```

Comment:
Why the `stringify`/`parse`?

The Task also expects the context (second parameter) to be the `SchematicContext` for the scheduling schematic.  That appears to be lost upon scheduling.  Is that an intentional API change?

Reply:
Want to make a pure JSON copy of it.

Reply:
The options are already being checked if they are a JsonObject above.  Turning a potentially large object into a string and then back again is not very efficient.